lib/src/video_player/method_channel_video_player.dart

```diff
@@ -401,7 +404,15 @@ class MethodChannelVideoPlayer extends VideoPlayerPlatform {
 
   @override
   Widget buildView(int? textureId) {
-    return Texture(textureId: textureId!);
+    if (defaultTargetPlatform == TargetPlatform.iOS) {
+      return UiKitView(
```

Comment:
Why did you change Texture to UiKitView. Does UiKitView is better than Texture?

Reply:
Does this change won't brake other stuff like PIP mode or notification of the player?

Reply:
Hi. 
> https://github.com/jhomlala/betterplayer/pull/488#discussion_r635503219 :
Why did you change Texture to UiKitView. Does UiKitView is better than Texture?

We migrate a native app using bitmovin to flutter. We start implementing a custom plugin to load bitmovin native sdk in iOS with UiKitView and Android with AndroidView. We detect that iOS part works fine, but with android verions prior to 10.0 it was displaying a black screen (similar to when you try to record the screen on a device playing drm video). Then we check this plugin and looked that working with Texture solved the android problem, but iOS was getting the black screen...
Then we implement a simple flutter plugin with fairlplay and UiKitView and verify that it was working fine on iOS. That's the reason we have change this. Maybe it can be changed to AndroidView too, but we didn't test it. We have more experience with iOS dev...

> https://github.com/jhomlala/betterplayer/pull/488#discussion_r635503537 :
Does this change won't brake other stuff like PIP mode or notification of the player?

We have try to make just the necessary changes to migrate from Texture to UiKitView. We maintain all the process, we just change the way AVPlayer is rendered, generating an UIView based on an AVPlayerLayer that get data from the AVPlayer. PIP process and notifications are related with the AVPlayer and this code is exactly the same. 
Also we have test that it's working fine on real devices (iPhons and iPads) and with production fairplay hls videos.

Maybe this is not the optimum way to implement (the plugin channel can be managed by each View and decentralize it) but it's the best way to maintain all the compatibility for Android and iOS with the fewest changes.